src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -304,6 +304,14 @@ internal virtual ImportChain ImportChain
             }
         }
 
+        internal virtual HashSet<string> QuickTypeIdentifierAttributeCheckSet
+        {
```

Comment:
Please add a doc comment. It wasn't clear to me until I scoured the implementation that "TypeIdentifierAttribute" is a type we are checking quickly for, rather than being part of the description of the function of this property.

Reply:
> Please add a doc comment.

Will do.